reaktive/src/jsMain/kotlin/com/badoo/reaktive/global/external/ExternalGlobal.kt

```diff
@@ -0,0 +1,3 @@
+package com.badoo.reaktive.global.external
+
+external val globalThis: dynamic
```

Comment:
Please make this internal.

Reply:
I cannot make internal, using the external keyboard to imply that it is not implemented by kotlin https://kotlinlang.org/docs/reference/js-interop.html#external-modifier

Reply:
What about `internal external val globalThis: dynamic`? The thing is we don't need this be part of the public API of the library.

Reply:
Done, amended. Thanks for your comments